DDG4/python/DDSim/DD4hepSimulation.py

```diff
@@ -447,8 +393,6 @@ def run(self):
       self._buildInputStage(geant4, actionList, output_level=self.output.inputStage,
                             have_mctruth=self._enablePrimaryHandler())
 
-    # ================================================================================================
```

Comment:
Can you limit to necessary changes only, no gratuitous beautyfications please? 
I am not sure if things are re-arrange a lot or `diff` gets confused.

Reply:
I was trying to :-)  The main reason for rearranging is that the worker actions needed to be separated from the master actions, as previously they were intermingled. That required some necessary rearranging.

Reply:
One way to help diff out could be to define helper functions in the order where things are in the file right now, and then use those helper functions in some new code that simply calls them (like `__setupWorker`). E.g. the UI setup can be put in a function at its current place in the file.